scripts/openfst/1.6.7/.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: generic
+
+matrix:
+  include:
+    - os: osx
+      osx_image: xcode8.2
+      compiler: clang
+    - os: linux
+      env: MASON_PLATFORM=linux
```

Comment:
No harm having `env: MASON_PLATFORM=linux` but it is also redundant/unneeded since this is the default.